server/embed/config.go

```diff
@@ -68,6 +68,11 @@ const (
 	StdErrLogOutput  = "stderr"
 	StdOutLogOutput  = "stdout"
 
+	// ExperimentalDistributedTracingAddress is the default collector address.
```

Comment:
I wasn't sure if this is the right place to put the constants? @gyuho any suggestions?

Reply:
Yes, here's good. Thanks!